src/main/scala/com/gu/acquisition/services/GAService.scala

```diff
@@ -69,9 +96,10 @@ private[services] class GAService(implicit client: OkHttpClient)
           "pa" -> "purchase", //This is a purchase
           "pr1nm" -> productName, // Product Name
           "pr1ca" -> conversionCategory.description, // Product category
-          "pr1pr" -> acquisition.amount.toString, // Product Price
+          "pr1pr" -> annualisedValue, // Product Price - we are tracking the annualised value here as this is what goes into the revenue metric
```

Comment:
On a general note, if we pull out the key values into an object with corresponding well named variables then the code is somewhat more self documenting and the clarifying comments in some cases aren't needed.

Reply:
In general I'd agree but I don't think it's worth introducing a new class here for what would essentially just be documentation. To be honest these comments are probably mostly not necessary, they were more to help me when I was writing the initial mappings and I left them in because in some cases they do add value.